### PR TITLE
#AV-39013: Configure SE dataNIC IP address as part of ansible-role-av…

### PR DESCRIPTION
**Pull request by chaitanyaavi**
…ise-csp role

### CODE REVIEW

File: library/avi_update_se_data_vnic_ip.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python
+#
+# @author: Gaurav Rastogi (grastogi@avinetworks.com)
```
Comment (grastogi23) [inline library/avi_update_se_data_vnic_ip.py]: change it to Chaitanya

File: library/avi_update_se_data_vnic_ip.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+#          Eric Anderson (eanderson@avinetworks.com)
+# module_check: supported
+#
+# Copyright: (c) 2017 Gaurav Rastogi, <grastogi@avinetworks.com>
```
Comment (grastogi23) [inline library/avi_update_se_data_vnic_ip.py]: change to Copyright: (c) 2017 Avi Networks Inc.

File: library/avi_update_se_data_vnic_ip.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+                }
+                vnic_networks_lst.append(vnic_networks)
+        if ipv6_addrs:
+            for ipv6_addr in ipv6_addrs.split(','):
+                addr, mask = ipv6_addr.split('/')
+                vnic_networks = {
+                    "ctlr_alloc": False,
+                    "ip": {
+                        "ip_addr": {
+                            "addr": addr.strip(),
```
Comment (grastogi23) [inline library/avi_update_se_data_vnic_ip.py]: how can user specify both ipv4 and ipv6 address?

**Comment by grastogi23** [comment]
should we also submit this module to the main repo?

**Comment by ericsysmin** [comment]
@chaitanyaavi assigned to you for edits

**Comment by grastogi23** [comment]
Need to handle support for the VLAN interfaces and vnic IP addresses.

**Comment by ericsysmin** [comment]
@grastogi23 once completed It should be in the main repo so that we can properly control versions and standardize installation

**Pull request closed without merging**